src/Microsoft.Identity.Web.OWIN/OwinTokenAcquisitionHost.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Net;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.Extensions.Options;
+using Microsoft.Identity.Client;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.Identity.Web.Hosts
+{
+    internal class OwinTokenAcquisitionHost : ITokenAcquisitionHost
+    {
+
+        IOptionsMonitor<MicrosoftIdentityOptions> _microsoftIdentityOptionsMonitor;
+        IOptionsMonitor<MergedOptions> _mergedOptionsMonitor;
+        IOptionsMonitor<ConfidentialClientApplicationOptions> _ccaOptionsMonitor;
+
+        public OwinTokenAcquisitionHost(
+            IOptionsMonitor<MicrosoftIdentityOptions> optionsMonitor, 
+            IOptionsMonitor<MergedOptions> mergedOptionsMonitor,
+            IOptionsMonitor<ConfidentialClientApplicationOptions> ccaOptionsMonitor)
+        {
+            _microsoftIdentityOptionsMonitor = optionsMonitor;
+            _mergedOptionsMonitor = mergedOptionsMonitor;
+            _ccaOptionsMonitor = ccaOptionsMonitor;
+        }
+
+        public Task<ClaimsPrincipal?> GetAuthenticatedUserAsync(ClaimsPrincipal? user)
+        {
+            return Task.FromResult<ClaimsPrincipal?>(null);
+        }
+
+        public string? GetCurrentRedirectUri(MergedOptions mergedOptions)
+        {
+            return null;
```

Comment:
>return null [](http://example.com/codeflow?start=12&length=11)

this should probably do something?

Reply:
yes, I agree. The web app part wasn't implemented yet.

Reply:
fixed.